language/stdlib/modules/LibraAccount.move

```diff
@@ -209,11 +208,7 @@ module LibraAccount {
         let above_threshold = approx_lbr_microlibra_value >= travel_rule_limit_microlibra;
         // travel rule only applies if the sender and recipient are both VASPs
         let both_vasps = VASP::is_vasp(sender) && VASP::is_vasp(payee);
-        // Don't check the travel rule if we're on testnet and sender
-        // doesn't specify a metadata signature
-        let is_testnet_transfer = Testnet::is_testnet() && Vector::is_empty(&metadata_signature);
```

Comment:
This is the one significant effect of this change: transactions > 1k Libra on testnet will now be subject to dual attestation.

@tzakian: this also means that once we go to implementing minting via transfers from a loaded DD account + address https://github.com/libra/libra/issues/4277, the max mint size will coincide with the dual attestation threshold.

Reply:
This sounds good to me, and makes sense :)